packages/blockchain-extension/extension/commands/deleteIdentityCommand.ts

```diff
@@ -66,7 +64,7 @@ export async function deleteIdentity(treeItem: IdentityTreeItem): Promise<void>
     } else {
         // Called from the tree
         const registryEntry: FabricWalletRegistryEntry = treeItem.registryEntry;
-        walletPath = registryEntry.walletPath;
+        wallet = await walletGenerator.getWallet(registryEntry);
```

Comment:
Do we need to check if this is undefined?

Reply:
check which bit is undefined?

Reply:
Can `wallet` be undefined, or will it always return a wallet?

Reply:
it will always return a wallet